pkg/plugins/golang/v4/scaffolds/internal/templates/cmd/main.go

```diff
@@ -195,11 +196,21 @@ func (f *MainUpdater) GetCodeFragments() machinery.CodeFragmentsMap {
 	}
 	if f.WireWebhook {
 		if f.IsLegacyPath {
-			setup = append(setup, fmt.Sprintf(webhookSetupCodeFragmentLegacy,
-				f.Resource.ImportAlias(), f.Resource.Kind, f.Resource.Kind))
+			if !f.MultiGroup || f.Resource.Group == "" {
+				setup = append(setup, fmt.Sprintf(webhookSetupCodeFragmentLegacy,
+					f.Resource.Version, f.Resource.Kind, f.Resource.Kind))
+			} else {
+				setup = append(setup, fmt.Sprintf(webhookSetupCodeFragmentLegacy,
+					f.Resource.ImportAlias(), f.Resource.Kind, f.Resource.Kind))
+			}
```

Comment:
We should only change the code when it is not legacy. Is there any reason why we do it?

Reply:
Thanks @camilamacedo86 
I got it
Revert it done